packages/aws-kms/src/index.ts

```diff
@@ -0,0 +1,70 @@
+import * as aws from '@aws-sdk/client-kms'
+import * as secp from '@noble/secp256k1'
+import * as crypto from '@atproto/crypto'
+import KeyEncoder from 'key-encoder'
+
+const keyEncoder = new KeyEncoder('secp256k1')
+
+export type KmsConfig = {
+  accessKey: string
+  secretKey: string
+  keyId: string
+  region: string
+}
+
+export class KmsKeypair implements crypto.DidableKey {
+  jwtAlg = crypto.SECP256K1_JWT_ALG
+
+  constructor(
+    private client: aws.KMS,
+    private keyId: string,
+    private publicKey: Uint8Array,
+  ) {}
+
+  static async load(cfg: KmsConfig) {
+    const client = new aws.KMS({
+      region: cfg.region,
```

Comment:
It's recommended by AWS to include `apiVersion` here, to avoid running into breaking changes to the KMS API.  Some more deets here: https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/locking-api-versions.html
```suggestion
      apiVersion: '2014-11-01',
      region: cfg.region,
```

Reply:
ah good looks 🙏